File: app/src/main/java/vn/loitp/app/activity/customviews/wwlmusic/utils/WWLMusicDataset.java
package vn.loitp.app.activity.customviews.wwlmusic.utils;

import com.utils.util.AppUtils;

import loitp.basemaster.R;

/**
 * Created by thangn on 2/27/17.
 */

public class WWLMusicDataset {
    public static DatasetItem[] datasetItems;

    static {
        int n = 50;
        datasetItems = new DatasetItem[n];
        for (int i = 0; i < n; i++) {
            datasetItems[i] = new DatasetItem(i + 1);
        }
    }

    public static class DatasetItem {
        public int id;
        public String title;
        public String subtitle;
        public String url;

        public DatasetItem(int _id) {
            String _url = "android.resource://" + AppUtils.getAppPackageName() + "/" + R.raw.vid_bigbuckbunny;

            this.id = _id;
            this.title = String.format("This is element #%d", _id);
            this.subtitle = "Pops Music";
            this.url = _url;
        }
    }
}

File: app/src/main/java/vn/loitp/app/activity/customviews/wwlmusic/utils/WWLMusicDataset.kt
package vn.loitp.app.activity.customviews.wwlmusic.utils

import com.utils.util.AppUtils.Companion.appPackageName
import vn.loitp.app.R

object WWLMusicDataset {

    @JvmField
    var datasetItems = ArrayList<DatasetItem>()

    class DatasetItem(id: Int) {
        var id: Int

        @JvmField
        var title: String

        @JvmField
        var subtitle: String

        @JvmField
        var url: String

        init {
            val tmpUrl = "android.resource://" + appPackageName + "/" + R.raw.vid_bigbuckbunny
            this.id = id
            title = String.format("This is element #%d", id)
            subtitle = "Pops Music"
            url = tmpUrl
        }
    }

    init {
        for (i in 0 until 50) {
            datasetItems.add(DatasetItem(i + 1))
        }
    }
}